src/pepr/operator/controllers/istio/istio-resources.ts

```diff
@@ -129,53 +131,56 @@ export async function istioEgressResources(
   namespace: string,
   generation: string,
   ownerRefs: V1OwnerReference[],
+  istioMode: string,
 ) {
   // Add needed service entries and sidecars if egress is requested
   if (hostResourceMap) {
-    // Validate existing egress gateway namespace and service
-    await validateEgressGateway(hostResourceMap);
-
-    // Add service entry for each defined host
-    for (const host of Object.keys(hostResourceMap)) {
-      // Create Service Entry
-      const serviceEntry = generateLocalEgressServiceEntry(
-        host,
-        hostResourceMap[host],
+    if (istioMode === IstioState.Ambient) {
+      // Validate existing egress waypoint namespace
+      try {
+        await validateNamespace(ambientEgressNamespace);
+      } catch (e) {
+        let errText = `Unable to get the egress waypoint namespace ${ambientEgressNamespace}.`;
+        if (e?.status == 404) {
+          errText = `Egress waypoint is not enabled in the cluster. Please enable the egress waypoint and retry.`;
```

Comment:
Is this error message trying to give feedback to the user that the `istio-egress-waypoint` component may not be deployed (i.e. the one that creates this namespace)?  If so, does it make sense to word this like so
```suggestion
          errText = `The '${ambientEgressNamespace}' namespace was not found. Ensure the 'istio-egress-waypoint' component is deployed and try again.`;
```

Reply:
updated